### DISCUSSION TITLE
Create issues for forks?

### DISCUSSION THREAD

**Discussion by RossBoylan** [community/community#23745] [New to GitHub]
<p>I created a fork and would like to be able to track issues on it for code specific to the branches I’m working on.  Is there a way to do that?  I don’t see any issues tab I can use (on my fork; I do on the upstream).</p>
<p><a href="https://github.com/RossBoylan/mccli" rel="nofollow noopener">https://github.com/RossBoylan/mccli</a> is where I’d like to create the issues.</p>
<p>I can think of some good reasons for wanting all the issues tracked through the master copy, but doing them locally (i.e., on the fork) would fit our current needs better.</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>You can enable/disable issues per repository, and it seems for forks they’re disabled by default. Go to the repository settings, scroll down a bit, and tick the “Issues” box:<br>
<img src="https://user-images.githubusercontent.com/5814919/181079201-e6b5419d-7b0c-4963-a1ac-28d99392bc89.png" alt="grafik" data-base62-sha1="zneQ7WDBwdvDMSoRmctNr0FhGHM" width="503" height="295"></p>

**Comment by RossBoylan** [comment]
<p>Thank you.  That worked great.</p>
<p>I had actually looked for a way to change the settings, but got hung up on the edit button that appears just below the settings icon.  It doesn’t do what I wanted, but thanks to your directions I found the right place.</p>
